scipy/stats/contingency.py

```diff
@@ -171,12 +171,12 @@ def chi2_contingency(observed, correction=True, lambda_=None):
 
     See Also
     --------
-    contingency.expected_freq
-    fisher_exact
-    chisquare
-    power_divergence
-    barnard_exact
-    boschloo_exact
+    scipy.stats.contingency.expected_freq
+    scipy.stats.fisher_exact
+    scipy.stats.chisquare
+    scipy.stats.power_divergence
+    scipy.stats.barnard_exact
+    scipy.stats.boschloo_exact
```

Comment:
This is a little unfortunate. The reason seems to be that we now have these functions in two public module listings (`stats` and `stats.contingency`), so for the `stats.contingency` submodule listing the reference don't resolve. I'd like to have another look later at if this is resolvable.

Reply:
Does not seem fixable unfortunately. Let's leave it as is.